cli/index.js

```diff
@@ -81,4 +431,425 @@ y.command({
+        },
+        userID: {
+            describe: 'An user ID',
+            demandOption: true,
+            type: 'string'
+        },
+        email: {
+            describe: 'An user email',
+            demandOption: true,
+            type: 'string'
+        },
+        newPassword: {
+            describe: 'The new password of the account',
+            demandOption: true,
+            type: 'string'
+        },
+        confirmPassword: {
+            describe: 'confirm the current password',
+            demandOption: true,
+            type: 'string'
+        },  
+    },
+    handler(argv) {
+        recover(argv)
+    }
+})
+
+// delete an account 
+y.scriptName("connectme")
+y.command({
+    command: 'delete',
+    describe: 'This can only be done by the logged in user',
+    builder: {
+        username: {
+            describe: 'Username',
+            demandOption: true,
+            type: 'string'
+        },
+    },
+    handler(argv) {
+        deleteuser(argv)
+    }
+})
+
+// get user by username
+y.scriptName("connectme")
+y.command({
+    command: 'get-user',
+    describe: 'get an user by username',
+    builder: {
+        username: {
+            describe: 'Username',
+            demandOption: true,
+            type: 'string'
+        },
+    },
+    handler(argv) {
+        getuser(argv)
+    }
+})
+
+// update account
+y.scriptName("connectme")
+y.command({
+    command: 'update-user',
+    describe: 'update & existen account',
+    builder: {
+        userId: {
+            describe: 'and userID',
+            demandOption: true,
+            type: 'string'
+        },
+        password: {
+            describe: 'password of the account',
+            demandOption: true,
+            type: 'string'
+        },
+        confirmPassword: {
+            describe: 'confirm the password',
+            demandOption: true,
+            type: 'string'
+        },
+        username: {
+            describe: 'and username',
+            demandOption: true,
+            type: 'string'
+        },
+        firstName: {
+            describe: 'your frist name',
+            demandOption: true,
+            type: 'string'
+        },
+        lastName: {
+            describe: 'your lastname',
+            demandOption: true,
+            type: 'string'
+        },
+        email: {
+            describe: 'your email',
+            demandOption: true,
+            type: 'string'
+        },
+        bio: {
+            describe: 'your biogaphy',
+            demandOption: true,
+            type: 'string'
+        },
+        website: {
+            describe: '...',
+            demandOption: true,
+            type: 'string'
+        },
+        location: {
+            describe: 'your actual location',
+            demandOption: true,
+            type: 'string'
+        },
+        picture: {
+            describe: '...',
+            demandOption: true,
+            type: 'string'
+        },Birthday:{

```

Comment:
inconsistent formatting and uppercase when everything else is a lower case

Reply:
done

Reply:
Nope, still cannot see the change.